frontend/src/components/SimpleRange/SimpleRange.js

```diff
@@ -1,18 +1,18 @@
-import { FormattedNumber } from 'react-intl'
 import { styled } from '@mui/material/styles'
 import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
-import Tooltip from '@mui/material/Tooltip'
 import PropTypes from 'prop-types'
 
 const GradientProgress = styled(LinearProgress)(({ theme, value }) => ({
   [`& .${linearProgressClasses.bar}`]: {
     overflow: 'hidden',
     backgroundColor: 'transparent',
+    borderRadius: 1,
     '&:before': {
       content: '""',
       display: 'block',
       width: '100%',
       height: '100%',
+      borderRadius: 1,
       transition: 'transform .4s linear',
       transform: `translateX(${100 - value}%)`,
       background: `linear-gradient(270deg, ${theme.palette.canary.main} 0%, ${theme.palette.chestnutRose.main} 50%, ${theme.palette.ultramarine.main} 100%)`, // eslint-disable-line
```

Comment:
The last bit of from my [previous review](https://github.com/CxSci/SA-dashboard/pull/198#discussion_r812358817) about Safari not rounding all ranges is still outstanding. Adding a borderRadius to GradientProgress fixed it for gradient ranges, but solid ranges still have the problem.

It's the separate ::before element of GradientProgress which fixes the rendering problem on Safari, so the solution that comes to my mind is to use the same solution for solid ranges by making GradientProgress into a function which takes `style` and `color` as inputs and change this line to something like ``background: style === 'solid' ? color : `linear-gradient...`'``.